rancher-ui-2.10.yaml

```diff
@@ -0,0 +1,94 @@
+package:
+  name: rancher-ui-2.10
+  version: "2.10.4"
+  epoch: 0
+  description: Rancher UI
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - bash
+      - busybox
+      - ca-certificates-bundle
+      - curl
+      - git
+      - npm
```

Comment:
These aren't used. `git-checkout` already depends on git, curl isn't used by anything, and we use yarn instead of npm

Reply:
I've removed `git`, and it seems other deps are required otherwise it results build failure.

Reply:
@EyeCantCU I had to revert to last stable commit since build somehow fails without including some build-time deps. Upstream build system might use those binaries under the hood.

Reply:
Built fine without them for me. pushed up